actionpack/test/dispatch/static_test.rb

```diff
@@ -143,6 +152,17 @@ def test_serves_gzip_with_propper_content_type_fallback
     assert_equal default_response.headers['Content-Type'], response.headers['Content-Type']
   end
 
+  def test_serves_files_with_access_control_headers
+    response = get("/foo/bar.html")
+
+    assert_equal 'http://rubyonrails.org',  response.headers["Access-Control-Allow-Origin"]
+    assert_equal 'X-Other-Custom-Header',   response.headers["Access-Control-Expose-Headers"]
+    assert_equal 'X-Another-Custom-Header', response.headers["Access-Control-Allow-Headers"]
+    assert_equal '42',   response.headers["Access-Control-Max-Age"]
+    assert_equal 'true', response.headers["Access-Control-Allow-Credentials"]
+    assert_equal 'GET',  response.headers["Access-Control-Allow-Methods"]
+  end
+
```

Comment:
Calling out Access-Control headers suggests that we built a feature to add these headers in particular. Using arbitrary/meaningless headers would make it clearer that we support any response headers.